Xamarin.Essentials/HapticFeedback/HapticFeedback.netstandard.tizen.tvos.watchos.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace Xamarin.Essentials
+{
+    public static partial class HapticFeedback
+    {
+        internal static bool IsSupported
+            => throw ExceptionUtils.NotSupportedOrImplementedException;
+
+        static async Task PlatformExecute(HapticFeedbackType type)
+        {
+            await Task.FromResult(0);
```

Comment:
Can you remove the async and this dummy task?

Reply:
I added everywhere "await Task.FromResult(0)" only because "await"is used on UWP. Tell me how best to remove it.